test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests.csproj

```diff
@@ -1,11 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) == ''">net7.0;net6.0</TargetFrameworks>
```

Comment:
@reyang Some of these projects have this funky `Condition="$(TARGET_FRAMEWORK) == ''"` syntax.

IIRC that is done so that you can specify the TFM from the command-line and it will override the project.

It seems to be utilized by the integration tests, ex: https://github.com/open-telemetry/opentelemetry-dotnet/blob/23b041c5539fdce770c40acc702de2a106a0de63/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/Dockerfile#L14

It doesn't look like we are bringing that bit forward on this PR. Seems OK to do as the builds are passing. My guess the reason it is working is because we're also passing `-f` (framework) to the CLI. Probably a better way to go about it 😄

Anyway I think if we're going to do this we should also remove the switches from the dockerfiles. We can do here or someone can do a follow-up just thought I would point it out.

Reply:
Good callout, I've cleaned it up a bit.